startserver.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+for command in pypy python python3.9 python3.8 python3.7 python3  
+do
+
+#!/bin/bash
+currentver=$($command -c "import platform;print(platform.python_version())")
+# echo $currentver
+requiredver="3.7"
```

Comment:
As of now, PyMine requires 3.7.9+, not any version of 3.7.

Reply:
i think this should have worked